extensions/amp-auto-ads/amp-auto-ads-spec.md

```diff
@@ -0,0 +1,268 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-auto-ads-spec"></a> `amp-auto-ads` spec
+
+## Description
+`amp-auto-ads` uses a remotely served configuration to dynamically inject ads
```

Comment:
Re-write as:

`amp-auto-ads` uses a remotely-served configuration file to dynamically inject ads into the page.